h/paginator.py

```diff
@@ -92,3 +92,7 @@ def wrapper(context, request):
             'page': page,
         }
     return wrapper
+
+
+def paginate_es(request, result, page_size=PAGE_SIZE):
```

Comment:
Yeah, this was sort of a placeholder, because in future we're gonna have to actually paginate the search results not just show a pagination control, that'll be more complicated, and presumably this function would get bigger. But it does seem pointless now